onnx/onnx.proto

```diff
@@ -124,7 +139,13 @@ message GraphProto {
 
   // A human-readable documentation for this graph. Markdown is allowed.
   optional string doc_string = 10;
-};
+
+  // Imported libraries are referenced as a collection of strings in the form of absolute
+  // URIs or relative paths. Where such relative paths are rooted is defined by tools and
+  // runtime implementations.
+  repeated string imported_libraries = 13;  
```

Comment:
Based on IRL discussion, this line is going away now, correct? (Or am I misunderstanding?)